cdap-explore/src/main/java/co/cask/cdap/hive/stream/HiveStreamInputFormat.java

```diff
@@ -96,7 +93,18 @@ public InputSplit createSplit(Path path, Path indexPath, long startTime, long en
                                                                StreamUtils.getGeneration(streamConfig));
 
     StreamInputSplitFinder.Builder builder = StreamInputSplitFinder.builder(streamPath.toURI());
-    return setupBuilder(conf, streamConfig, builder).build(SPLIT_FACTORY);
+
+    // Get the Hive table path for the InputSplit created. It is just to satisfy hive. The InputFormat never uses it.
```

Comment:
Updated the javadoc for StreamInputSplit to include the detail of how *funny* Hive behaves.